docs/tutorials/fundamentals/installation.md

```diff
@@ -169,119 +166,142 @@ you're not using.
+![image: napari Software License Agreement verbage](.../docs/images/bundle_06.png)
 
-After you drag the `napari` icon onto the `Applications` icon napari will be
-installed in your Applications folder and available for opening by double
-clicking on it.
+After reading this agreement, click ‘Continue’ to be prompted to agree to the Software License Agreement in order to proceed with installation.
 
-![image: MacOS Applications folder with
-napari](../assets/tutorials/installation/bundle_applications_folder.png)
+![image: Prompt to agree to napari Software License Agreement](.../docs/images/bundle_07.png)
 
-The first time you try and open napari you will get the "unverified developer
-warning" that appears below:
+On the following page, you will be shown how much space the installation will use and can begin the standard installation by clicking ‘Install.’
 
-![image: MacOS developer
-warning](../assets/tutorials/installation/bundle_dev_warning.png)
+![image: napari installer space requirement](.../docs/images/bundle_09.png)
 
-Don't be alarmed, this is because the napari team has not yet obtained a
-developer certificate from Apple, but we will be doing this promptly. It is safe
-to click "Cancel". To get napari to open you must enter "Systems Preferences"
-and click the "Security & Privacy" icon, circled in red below:
+However, if you would like to change the install location, you may specify a different location by clicking ‘Change Install Location…’ and following the subsequent prompts before starting the installation.
 
-![image: MacOS Security & Privacy
-settings](../assets/tutorials/installation/bundle_security_privacy.png)
+The installation progress can be monitored on the following window.
 
-Once inside the "Security & Privacy" tab, you should see a message about napari
-being blocked and an "Open Anyway" button, circled in red below:
+![image: napari installer progress monitoring page](.../docs/images/bundle_10.png)
 
-![image: MacOS "Open Anyway"
-settings](../assets/tutorials/installation/bundle_open_anyway.png)
+If installation is successful, you will see the window shown below and you may now close the installation wizard and move it to trash.
 
-After clicking this button you'll get one final warning about napari not being
-verified, but now you'll have an "Open" button, as seen below:
+![image: napari installer success page](.../docs/images/bundle_11.png)
 
-![image: MacOS open bundle dialog
-box](../assets/tutorials/installation/bundle_open.png)
+You can now get started using napari! Use Launchpad to open the application. 
 
-After clicking "Open", the viewer should appear. Don't worry, you only have to
-go through this process once when you install a new bundle.
+![image: napari icon in MacOS laundpad](.../docs/images/bundle_13.png)
+
+```{note} 
+The first time you open napari you must use the Launchpad, but subsequently, the napari application will show up in Spotlight search.
```

Comment:
```suggestion
The first time you open napari you must use the Launchpad, but subsequently, the napari application should show up in Spotlight search.
```

I am not sure if it happens immediately after one usage, or if it's more about macOS learning about your preferences (e.g. it requires more than one click to appear in Spotlight).